add wordle extension

chrisalxlng opened the pull request:
## Description

This is a new extension that allows users to play Wordle offline in Raycast. It currently supports English and German word sets. It has 3 commands:

- `Play English (US)`: Play Wordle with an English word set
- `Play German (DE)`: Play Wordle with a German word set
- `Show History`: Show summaries of past puzzles

## Screencast

https://user-images.githubusercontent.com/65340138/217403793-c730cbf2-00c4-4dca-abfd-a1147b5c4558.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer commented on extensions/wordle/package.json:
```diff
@@ -0,0 +1,54 @@
+  "commands": [
+    {
+      "name": "play_en-US",
+      "title": "Play English (US)",
```
```suggestion
      "title": "Play English (US)",
      "subtitle": "Wordle",
```

chrisalxlng replied:
added subtitle

pernielsentikaer commented on extensions/wordle/package.json:
```diff
@@ -0,0 +1,54 @@
+    },
+    {
+      "name": "play_de-DE",
+      "title": "Play German (DE)",
```
```suggestion
      "title": "Play German (DE)",
      "subtitle": "Wordle",
```

chrisalxlng replied:
added subtitle

pernielsentikaer commented on extensions/wordle/package.json:
```diff
@@ -0,0 +1,54 @@
+    },
+    {
+      "name": "show_history",
+      "title": "Show History",
```
```suggestion
      "title": "Show History",
      "subtitle": "Wordle",
```

chrisalxlng replied:
added subtitle

pernielsentikaer reviewed (changes requested):
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* It seems like it's case-sensitive, which might confuse everyone 😄

https://user-images.githubusercontent.com/38033107/217859286-43a8bffd-01b4-407c-9db7-fd22388576c5.mp4

* Maybe also add a Delete Summary action

![Raycast-1tkwV3G4@2x](https://user-images.githubusercontent.com/38033107/217859481-f93e063f-4bfa-4942-b6d5-b06d7735be08.png)

- Is it possible to change the action so it matches the one from the dropdown if de is selected?

![Raycast-PmfA5VVG@2x](https://user-images.githubusercontent.com/38033107/217859926-ac1db2f3-1fb2-4e04-afab-1c44b442599e.png)


I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

chrisalxlng commented:
Hey👋
Thank you for your feedback. I have addressed all the issues/suggestions you mentioned:

1. Resolve Case Sensitivity
<img width="1000" alt="wordle 2023-02-10 at 00 16 48" src="https://user-images.githubusercontent.com/65340138/217961224-40e7ca1a-14f6-4949-b808-de434a252aef.png">

2. Add Delete Summary action
<img width="1000" alt="wordle 2023-02-10 at 00 18 16" src="https://user-images.githubusercontent.com/65340138/217961403-e412a4cc-72e6-4a7c-bf9f-f05bdc03bfee.png">

3. Set `Start Playing` action of selected language as first action
<img width="1000" alt="wordle 2023-02-10 at 00 17 45" src="https://user-images.githubusercontent.com/65340138/217961609-ae88b671-8538-4c47-83eb-3b7a1ce9709c.png">
<img width="1000" alt="wordle 2023-02-10 at 00 17 39" src="https://user-images.githubusercontent.com/65340138/217961619-6164fec5-05f2-4c3c-a461-1042a5f6e66c.png">

Thanks again for taking the time reviewing my extension!

pernielsentikaer reviewed (approved):
Hi 👋

Looks good to me, approved 🔥
